src/app/medicine/medicine.cpp

```diff
@@ -1,6 +1,6 @@
 #include "medicine.hpp"
 
-Medicine::Medicine(const std::string& name, const double price, const int amount, const int expiration_date,
+Medicine::Medicine(const std::string& name, const double& price, const int& amount, const int& expiration_date,
```

Comment:
We spoke about & with such types that is not needed.
However if I think about it now, it's just a good pratice to remember about it even if that does not provide any value.